tdrs-backend/tdpservice/reports/views.py

```diff
@@ -1,27 +1,73 @@
 """Check if user is authorized."""
 import logging
+from boto3 import client
 
+from django.conf import settings
 from rest_framework import mixins, viewsets
+from rest_framework.response import Response
 
 from ..users.permissions import CanUploadReport
+
+from .serializers import ReportFileSerializer, PresignedUrlInputSerializer
 from .models import User
-from .serializers import ReportFileSerializer
+
+from rest_framework.decorators import action
```

Comment:
nit: import organization

From [PEP8](https://www.python.org/dev/peps/pep-0008/#imports):

> Imports should be grouped in the following order:
>
> Standard library imports.
> Related third party imports.
> Local application/library specific imports.
> You should put a blank line between each group of imports.

> Absolute imports are recommended, as they are usually more readable and tend to be better behaved (or at least give better error messages) if the import system is incorrectly configured

Additionally, alphabetization by module name is a bonus :D